medusa/service/grpc/medusa.proto

```diff
@@ -42,9 +42,12 @@ message BackupStatusRequest {
 }
 
 message BackupStatusResponse {
+  // this response is about a _node_ backup. The fields about nodes are not correct here
+  // we should deprecate/remove them but that breaks backwards compatibility
```

Comment:
Thought:
Could we introduce a new response and deprecate the old one? If that's feasible, maybe create a refactor ticket.

Reply:
Yes, I agree with this. We initially wanted to preserve backwards compatibility.
But I checked with @adejanovski and we decided to just drop this straight away.
I've pushed an ammended commit that drops this (and it also does some cleanup after rebasing on top of the other PRs).